scripts/inspect_threshold_edge_haplotypes.py

```
@@ -0,0 +1,316 @@
+def main() -> None:
+    import os
+
+    os.environ["PYSPARK_SUBMIT_ARGS"] = "--driver-memory 8g --executor-memory 8g pyspark-shell"
+    hl.init(quiet=True)
+
+    ht_variants = hl.read_table(f"{BASE}.variants.ht")
+    ht_hap_ac = hl.read_table(f"{BASE}.hap_ac.ht")
+    ht_parents = hl.read_table(f"{BASE}.parents.ht")
+    try:
+        ht_final = hl.read_table(f"{BASE}.ht")
+    except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)